index.Rmd

```diff
@@ -28,15 +28,16 @@ This project explores the number and frequency of updates in a project currently
 
 ## Methods
 
-mon amour - can you write the section on how you extracted the data?
+The [dependency manifest file](https://github.com/canada-ca/tracker/blob/003a3d6e0452fb9a248a7d51fd5556bc03dfbcc6/frontend/package.json) from the frontend of Government of Canada's Tracker project was used as a representative example of a modern software project.
 
-On October 22, 2020, the registry for NPMJS...
+Using the following command, package names (both development and production) were extracted from the `package.json` and details for each was retrieved from the npm API.
```

Comment:
It is good practice to report the date on which data was extracted. Do you mind reviewing to include it in your description?

Reply:
If the date is to act as a version the file, the SHA in the GitHub perma-link actually refers to the manifest file as it exists in that specific commit: 
`https://github.com/canada-ca/tracker/blob/003a3d6e0452fb9a248a7d51fd5556bc03dfbcc6/frontend/package.json`
I think that achieves the goal, just in a different form.

If we are thinking about the version of the stuff returned from the api... that's pretty stable and would need a `/v2/` in the url or something if it were to change.

I'm happy to add the date if that's a stylistic thing/convention you want to follow, but if the concern is reproducability I think that aspect is handled. 
Thoughts?

Reply:
It is for reproducibility - if someone wants to reproduce this analysis they should be able to do so by following the methodology - if we need to add more steps then let's do so - if those include a commit, then I agree that it would serve as a version control so the person could download the same version as the one used in the analysis. I leave it to you to expand on this as you see fit. Thanks.

Reply:
OK, so I added another commit that downloads the data reproducibly and replaces the existing `data/raw/nodejs.csv` with one that is exactly what was produced from the package.json and placed it in `data/processed/nodejs.csv`. The text has been updated to reflect that.

Reply:
LGTM